.github/pull_request_template.md

```diff
@@ -4,6 +4,11 @@
 
 
 ## How was the functionality tested and verified?
+All steps should be completed in the order presented, and you should not move on to the next step until the previous is completed.
+- [ ] The title of the `html` page is the same as the chart.
+- [ ] The `meta_data.json` file is filled in correctly (the description is a complete sentence, the title is the same as the visual).
+- [ ] The visual is styled similar to other charts (centered title/subtitle, no legends unless absolutely necessary, etc.)
```

Comment:
at some point, I think it would be nice to have a more explicit style guide to link here